drivers/hmis/spec/models/hmis/hud/enrollment_spec.rb

```diff
@@ -227,4 +227,122 @@
       end
     end
   end
+
+  describe 'occurrence point form instances' do
+    let(:role) { :OCCURRENCE_POINT }
+    let!(:project) { create(:hmis_hud_project, data_source: ds1) }
+    let!(:hoh_enrollment) { create(:hmis_hud_enrollment, project: project, data_source: ds1, household_id: 'household1', relationship_to_hoh: 1) }
+    let!(:spouse_enrollment) { create(:hmis_hud_enrollment, project: project, data_source: ds1, household_id: 'household1', relationship_to_hoh: 3) }
+
+    context 'no relevant instance exists' do
+      it 'does not return the form when no instance exists' do
+        expect(hoh_enrollment.occurrence_point_forms).to be_empty
```

Comment:
In this test, is there any OCCURRENCE_POINT form definition in the database? I'm not totally understanding asserting about "the form" when there is no form (as far as I can see in the fixtures)